plugins/CoreHome/CoreHome.php

```diff
@@ -379,7 +378,8 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = 'CoreHome_ShortcutWebsiteSelector';
         $translationKeys[] = 'CoreHome_ShortcutCalendar';
         $translationKeys[] = 'CoreHome_ShortcutSearch';
-        $translationKeys[] = 'CoreHome_ShortcutHelp';
+        $translationKeys[] = 'CoreHome_ShortcutSearch';
```

Comment:
Seems you have duplicated `Search` and removed `Help` on accident 🤷 
```suggestion
        $translationKeys[] = 'CoreHome_ShortcutHelp';
```